src/stories/buttongroup.stories.js

```diff
@@ -136,7 +82,7 @@ export const CheckboxButtonGroup = () => html`
     ></cod-form-check>
   </cod-button-group>
   <br /><br />
-  <cod-button-group data-type="group" data-label="basic example">
+  <cod-button-group data-type="group" label="basic example">
```

Comment:
We still have to use `data-label` here since this is for the button group component which we haven't fixed yet.